### PR TITLE
Issue 109

### LINKED ISSUE #388: [109] Issue #109 - May 17, 2018
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/master/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

### PR DESCRIPTION
**Pull request by BasThomas**
Closes #388

### CODE REVIEW

File: _posts/2018-05-17-issue-109.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- [SR-7629](https://bugs.swift.org/browse/SR-7629) [Compiler] Wrong fixit when the type doesn't conform to a public protocol
+
+{% include task_remind.html %}
+
+### Swift Unwrapped
+
+In [episode 57](https://spec.fm/podcasts/swift-unwrapped/141585), Jesse and JP discuss the Swift for TensorFlow Design Overview that we mentioned two weeks ago.
+
+In [episode 58](https://spec.fm/podcasts/swift-unwrapped/144196), Jesse and JP discuss the reimplentation of Implicitly Unwrapped Optionals (IUOs) in Swift, and what that brings us.
+
```
Comment (jessesquires) [inline _posts/2018-05-17-issue-109.md]: both looks great! 👍

File: _posts/2018-05-17-issue-109.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+### News and community
+
+[Nicole Jacque])(https://github.com/najacque/) and [Mishal Shah](https://twitter.com/mishaldshah/) announnced [a community hosted Swift continuous integration](https://twitter.com/SwiftLang/status/992166029613674497) to make it easier to support more platforms for Swift.
```
Comment (jpsim) [inline _posts/2018-05-17-issue-109.md]: s/announnced/announced/

File: _posts/2018-05-17-issue-109.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+[Ted Kremenek](https://twitter.com/tkremenek) announced that Swift for TensorFlow [will be developed](https://forums.swift.org/t/swift-for-tensorflow-to-be-developed-on-tensorflow-branch-on-apple-swift-on-github/12595) on a `tensorflow` branch on apple/swift, allowing closer collaborations among the Swift and TensorFlow teams.
+
+[Brian Gesiak](https://twitter.com/modocache) wrote [an article](https://modocache.io/llvm-memory-buffer) on how Swift and Clang use LLVM to read files into memory.
+
+### Commits and pull requests
+
+[Doug Gregor](https://github.com/DougGregor) merged [a pull request](https://github.com/apple/swift/pull/16563) that eliminates more of `SubstitutionList`. In a followup [pull request](https://github.com/apple/swift/pull/16568) all remaining `SubstitutionList`s have been removed. "Introduced during the bring-up of the generics system in July, 2012, `Substitution` (and `SubstitutionList`) have been completely superseded by `SubstitutionMap`."
```
Comment (jpsim) [inline _posts/2018-05-17-issue-109.md]: Consider making the quote here a markdown quote.

File: _posts/2018-05-17-issue-109.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+> Related Projects includes access to specific sub-categories that are dedicated to projects within the Swift community and are separate from the Swift language itself. This new section of Swift Forums is launching today with support for a number of projects, including:
+>
+> - Kitura
+> - SourceKitten
+> - SwiftLint
+> - SwiftNIO
+> - SwiftProtobuf
+> - Vapor
+
+[Michael Ilseman](https://github.com/milseman) pitched [a proposal](https://forums.swift.org/t/pitch-character-and-string-properties/11620) to increase the usefulness of `Character` and approachability of programing in Swift.
```
Comment (jpsim) [inline _posts/2018-05-17-issue-109.md]: s/programing/programming/

**Comment by BasThomas** [comment]
Thanks for reviewing @jpsim!

**Comment by jessesquires** [comment]
@BasThomas you can have Sublime do spellcheck 😊 

open preferences, add:

```
"dictionary": "Packages/Language - English/en_US.dic",
"spell_check": true,
```

**Comment by BasThomas** [comment]
Ooooh that helps a lot, thanks!

**Pull request merged**